CBLClient/Apps/CBLTestServer-Dotnet/TestServer/ValueSerializer.cs

```diff
@@ -117,7 +125,8 @@ internal static object DeserializeValue(String value)
             }
             else if (value.StartsWith("\"") && value.EndsWith("\""))
             {
-                return value.Replace("\"", String.Empty);
+                string trimVal = value.Substring(1, value.Length - 2);
+                return trimVal;
```

Comment:
do you want to 
`return value.Stustring(1, value.length - 2)`
?

Reply:
In java and other languages we are using  return (T) value.substring(1, value.length() - 1);, But in dot net  -2  is exclusive Ex:  String a = "g=nhz";
		Console.WriteLine(a.Substring(1, a.Length-2));    OUTPUT  =nh

Reply:
i mean to combine the 2 lines into 1, trimVal seems not needed. 
I am not questioning the index value

Reply:
If string is " a"lkl"l " previous  code returns alkll, where as current code returns a"lk"l

Reply:
I think Eunice meant to combine line #128 and #129. She is referring to new code, not old code. new code is fine, you return the value directly as it is one line.